.cirrus.yml

```diff
@@ -175,7 +175,7 @@ task:
     - ccache --show-stats
   check_script:
     - src\test_bitcoin.exe -l test_suite
-    - src\bench_bitcoin.exe --sanity-check > NUL
+    - src\bench_bitcoin.exe -sanity-check -priority-level=high
```

Comment:
is there a reason why we changed this to only include `-priority-level=high`?

Reply:
See https://github.com/bitcoin/bitcoin/pull/26158#discussion_r1019382276

Reply:
unrelated: It could make sense to have one CI task run all priorities

Reply:
Ok. The last commit has been dropped.

Reply:
unrelated: It could be set to run all priorities explicitly (and mentioned in the CI task name) to make it explicit and documented for devs.